### ISSUE TITLE
Can't assign Categories without the edit_posts permission

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Capabilities#40]
@agapetry We chatted about this scenario in Slack

Essentially, you need the edit_posts permission to assign any taxonomy terms, even if the taxonomy is attached to a different post type

**Comment by agapetry** [comment]
@stevejburge Correct. It would be possible to develop a workaround if this poses a major problem for anyone.
